rockpaper.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Rock Paper Scissors</title>
+        <meta charset="UTF-8"/>
+    </head>
+    <body>
+        <script src="./script.js"></script>
+    </body>
+</html>
```

Comment:
Always add an empty line at the end of every file

Reply:
fixed